### ISSUE TITLE
bitbucket project is gone, do you still have the zip files?

### ISSUE THREAD

**Issue by ec1oud** [yshurik/inferno-rpi#1]
The links are broken.

https://bitbucket.org/infpi/inferno-rpi/downloads/

**Comment by ec1oud** [comment]
Maybe there should also be complete build instructions to generate the image.

**Comment by yshurik** [comment]
I will check, it should be somewhere, which file you are looking for?

**Comment by ec1oud** [comment]
The latest zip.  The latest one you mentioned in your blog was 0.6:  http://lynxline.com/inferno-rpi-0-6-release/  but maybe you have a newer one?

Why not restore all the files somewhere?  I'd volunteer to keep them available via IPFS or bittorrent.  But github can store them too, I think.

I should learn how to reproduce the image anyway, if that's within reach.

I'm trying to understand why plan9 is seeing regular improvement (9front and others) while inferno is apparently much less popular, even though it was supposed to be the next step forward, has a better GUI, and so on.

**Comment by yshurik** [comment]
Added into releases: https://github.com/yshurik/inferno-rpi/releases/tag/v0.6

**Comment by ec1oud** [comment]
Thanks.

So I dug out the original Raspberry Pi (model B) and booted it up to see what was on the SD card... guess what, it booted to Inferno.  It looks like I tried this in 2018 and forgot (judging from the date on the zip file, which I also found).  But thanks for the info about loading kernel.bin to get the graphical shell.  And it's good to have the zip back online in case someone else wants to try it.

It doesn't work with either of my keyboards, but I can use the mouse, start programs from the menu etc.  Maybe it's something about the keyboards themselves.
